packages/components/organisms/f-takeawaypay-activation/src/tenants/en-GB.js

```diff
@@ -2,8 +2,9 @@ const messages = {
     locale: 'en-GB',
 
     messages: {
-        titleActivationAvailable: 'Welcome back. Let\'s activate Takeaway Pay on your account.',
-        titleActivationNotLoggedIn: 'Let\'s get you started with Takeaway Pay',
+        titleActivationAvailable1: 'Welcome back, ',
+        titleActivationAvailable2: 'Let\'s activate Takeaway Pay on your account.',
```

Comment:
This can be combined into one prop like 
```suggestion
        titleActivationAvailable: 'Welcome back, {name}. Let’s activate Takeaway Pay on your account.',
```
And then you won't need a `welcomeText`computed prop and just pass the value like:
```
 <h1 :class="$style['c-takeawaypayActivation-title']">
      {{ $t('messages.titleActivationAvailable', { consumerGivenName }) }}
 </h1>
```

Reply:
@dandel10n Thanks, great idea. Changed according to suggestion.